src/pretix/plugins/checkinlists/exporters.py

```diff
@@ -93,10 +93,17 @@ def _get_queryset(self, cl, form_data):
         ).order_by().values('position_id').annotate(
             m=Max('datetime')
         ).values('m')
+
+        cqs2 = Checkin.objects.filter(
```

Comment:
We should try to also eliminate this sub-query

Reply:
For any specific reason? Subqueries aren't inherently bad.

We should change it though, see below.